reqwest-tracing/Cargo.toml

```
@@ -14,7 +14,7 @@ opentelemetry_0_20 = ["opentelemetry_0_20_pkg", "tracing-opentelemetry_0_21_pkg"
 opentelemetry_0_21 = ["opentelemetry_0_21_pkg", "tracing-opentelemetry_0_22_pkg"]
 opentelemetry_0_22 = ["opentelemetry_0_22_pkg", "tracing-opentelemetry_0_23_pkg"]
 opentelemetry_0_23 = ["opentelemetry_0_23_pkg", "tracing-opentelemetry_0_24_pkg"]
-
+deprecated-fields = []
```

Let's use an `_` here to be consistent with the styling in the code. Let's also use the term "attributes" to match the name used for fields in the [opentelemetry standard](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/trace/semantic_conventions/http.md#common-attributes)

```suggestion
deprecated_attributes = []
```